composed/uniffi/src/composed.udl

```diff
@@ -0,0 +1,25 @@
+namespace composed {
+};
+
+[Error]
+enum Error {
+    "Common",
+    "External",
+};
+
+[ExternalInterface="base"]
+typedef extern WrappingClass;
```

Comment:
This doesn't work. [Documentation](https://mozilla.github.io/uniffi-rs/udl/ext_types_external.html#external-interface-types) says it does.